firmware/application/apps/capture_app.hpp

```diff
@@ -67,7 +67,7 @@ class CaptureAppView : public View {
     uint32_t sampling_rate = 0;
     uint32_t anti_alias_baseband_bandwidth_filter = 2500000;  // we rename the previous var , and change type static constexpr to normal var.
 
```

Comment:
Add settings too?

Reply:
Apps like capture app and audio app are tendencious.
If app settings are used, you loose the 'go to tuned Freq from calling app' feature (used like in Recon and Looking Glass)

Reply:
Hmm -- audio would have the same problem then.
One of my next planned changes Is to figure out a way to declare the settings you actually want to load. Maybe we could use that. The main problem I need to solve is that each app needs to have some radio settings reset to a sane default and I was hoping to reuse SettingsManager for that. I'll sort it out in another PR then.

Reply:
I think a better approach for "calling app freq" would be to have the caller explicitly set the frequency via a method call. Again, something for future PR.